media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -3,16 +3,17 @@
 
 EAPI=8
 
-CMAKE_REMOVE_MODULES_LIST=( FindFreetype )
-LUA_COMPAT=( luajit )
-PYTHON_COMPAT=( python3_{9..12} )
+CMAKE_REMOVE_MODULES_LIST=(FindFreetype)
+LUA_COMPAT=(luajit)
+PYTHON_COMPAT=(python3_{9..12})
 
-inherit cmake lua-single optfeature python-single-r1 xdg
+inherit desktop cmake lua-single optfeature python-single-r1 xdg
```

Comment:
Does desktop need to be before cmake here?

If I remember right, there are cases when that is true, but usually these are also listed in an alpabetical order.

Reply:
No need to be before I gonna change to alphabetical order